src/ProofOfConcepts/connectors/connectors.tsx

```diff
@@ -0,0 +1,361 @@
+import React from "react";
+import {
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction,
+  Flex,
+  FlexItem,
+  Label,
+  LabelGroup,
+} from "@patternfly/react-core";
+import PlusCircleIcon from "@patternfly/react-icons/dist/esm/icons/plus-circle-icon";
+import OutlinedQuestionCircleIcon from "@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon";
+
+export const ConnectorTypeListItem: React.FunctionComponent = () => {
```

Comment:
Maybe call this component the list, such as `ConnectorTypeListItem` and then it'd then contain `ConnectorTypeListItem`s.  It'd be worth refactoring out the `DataListItem` elements into it's own component that you can then pass the needed properties to.  Here's an example of what I mean, [this is the old component](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/app/components/ConnectorTypeCard/ConnectorTypeCard.tsx) that this POC could be replacing.

Reply:
Would you mean creating a props component and then also having the data list component separate?

Reply:
Sorry, I didn't catch this earlier :smile: 

See if you can take one of the `DataListItem` elements underneath this `DataList` you've created and turn it into it's own component that accepts properties for each of the items it needs to display.